File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        int len=s.length -1;
        char temp;
        for(int i=0;i<len;i++){
            temp=s[i];
            s[i]=s[len];
            s[len--]=temp;
        }
    }
}

File: 0344-reverse-string/0344-reverse-string.cs
public class Solution {
    public void ReverseString(char[] s) {
        int len = s.Length-1;
        for(int i=0;i<s.Length/2;i++)
        {
            char temp = s[i];
            s[i] = s[len];
            s[len] = temp;
            len--;
        }
    }
}